ocs_ci/ocs/node.py

```diff
@@ -1754,6 +1761,29 @@ def verify_all_nodes_created():
         )
 
 
+def power_on_vms():
```

Comment:
What about rename to something like power_on_ocp_node_vms()

And mention to docstring that it makes sure all cluster node VMs are powered on.

Reply:
done